webapp/public/index.html

```
@@ -0,0 +1,351 @@
+    <div class="container nav-inner">
+
+      <!-- Logo -->
+      <a class="nav-logo" href="#" aria-label="Career Ops home">
+        <div class="nav-logo-mark" aria-hidden="true">C</div>
+        <span class="nav-logo-name">Career<span>Ops</span></span>
+      </a>
+
+      <!-- Tabs -->
+      <ul class="nav-tabs" role="tablist" aria-label="Main navigation">
+        <li role="presentation">
+          <button class="nav-tab active" role="tab" aria-selected="true"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)